docs/docs-beta/docs/guides/deployment/docker.md

```diff
@@ -1,3 +1,197 @@
 ---
-title: "Deploying to Docker"
+title: "Deploying with Docker"
+description: A guide to deploying Dagster with Docker.
 ---
+
+This guide provides instructions for deploying Dagster using Docker. It's common to use Docker containers to run the Dagster web server, daemon, and code servers. These containers are typically deployed using AWS ECS or another container-based orchestration system.
+
+## What you'll learn
+
+- The different Docker containers that run as part of a Dagster Docker deployment
+- How to define Docker images for each of these containers
+- How to write a docker-compose file that stands up these containers
+
+<details>
+  <summary>Prerequisites</summary>
+- Familiarity with Docker
```

Comment:
Do any files like `dagster.yaml` need to be in the Dagster project to do this?

Reply:
Good point - added